opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -603,17 +603,12 @@ namespace Opm {
                     properties.dropInjectionControl(WellInjector::THP);
 
                 /*
-                  What a mess; there is a sensible default BHP limit
-                  defined, so the BHPLimit can be safely set
-                  unconditionally - but should we make BHP control
-                  available based on that default value - currently we
-                  do not do that.
+                  There is a sensible default BHP limit defined, so the BHPLimit can be
+                  safely set unconditionally, and we make BHP control based on that default value.
```

Comment:
Please comment here that it is difficult to infer this form the manual, but that the (new) current behavior seems to agree with the behavior of ECLIPSE.